docs/walk-throughs/4-listener-rfc-callback.md

```
@@ -0,0 +1,367 @@
+
+- *Goto* → *Expert Functions* → *External Security* → *Maintain ACL Files*.
+- Edit **reginfo**. Add above any deny-all rule at the bottom:
+
+```text
+P TP=TEST_LISTENER HOST=* ACCESS=* CANCEL=*
+```
+
+- Save.
+- *Reload ACL Files* (this is critical — SAP does *not* hot-reload it).
+
+![](./resources/recordings/edit_reginfo_sap.gif)
```

markdownlint-cli2 (0.22.1)

[warning] Images should have alternate text (alt text)

(MD045, no-alt-text)